src/server/chat/messages.js

```diff
@@ -15,7 +15,7 @@ const createAttributesObject = function(messageContent) {
 		}
 
 		if (!skinId) {
-			skinId = skins.idList.includes(messageSections[i]) ? messageSections[i] : undefined;
+			skinId = skins.idList[messageSections[i] ? messageSections[i] : undefined];
```

Comment:
Not sure what the logic is here? Currently it checks whether `messageSections[i]` resolves to `true`, not whether it exists in the list.

Reply:
Just to add: It works, but it looks accidental?

Reply:
I'm pretty sure this was brilliant when I wrote it, but I agree it looks kind of confusing. Not sure if I should change it, any suggestions? I could also just add some comment to clarify it.

Reply:
How about `skinId = messageSections[i] ? skins.idList[messageSections[i]] : undefined;`?

Reply:
Checked again just in case and `messageSections[i]` will always be a string, so we don't need to check whether it's undefined.

So technically, just `skinId = skins.idList[messageSections[i]];` will suffice! Non-existing skins will return `undefined`. But since that might be a bit confusing, it could probably do with a comment in that case.